Misc/NEWS.d/next/Library/2019-09-16-09-54-42.bpo-38136.MdI-Zb.rst

```diff
@@ -0,0 +1,3 @@
+Changes AsyncMock call count and await count to be two different counters.
```

Comment:
What about docs updated?

Reply:
This comment was poorly placed.
I didn't mean to ask that the doc updates should be included in the news entry -- I was asking why I didn't see any changes in the documentation to reflect the changes you did in the code.

Reply:
Oh sorry that makes more sense. I just added something to the assert_awaited() call, I'm not sure if that is enough. Would a full example on the example page be overkill or a good idea?